java/src/jmri/jmrit/beantable/LogixTableAction.java

```diff
@@ -1493,7 +1493,7 @@ boolean checkConditionalReferences(String logixName) {
                         String[] msgs = new String[]{c.getUserName(), c.getSystemName(), cRef.getUserName(),
                             cRef.getSystemName(), xRef.getUserName(), xRef.getSystemName()};
                         JOptionPane.showMessageDialog(null,
-                                Bundle.getMessage("LogixError11", msgs), // NOI18N
```

Comment:
Is there a need to create the String[] just to pass it in? The Bundle call could just pass in individual elements.